Managers/AudioMan.cpp

```diff
@@ -602,6 +602,11 @@ namespace RTE {
 			result = (result == FMOD_OK) ? channel->setPriority(soundContainer->GetPriority()) : result;
 			float pitchVariationMultiplier = pitchVariationFactor == 1.0F ? 1.0F : RandomNum(1.0F / pitchVariationFactor, 1.0F * pitchVariationFactor);
 			result = (result == FMOD_OK) ? channel->setPitch(soundContainer->GetPitch() * pitchVariationMultiplier) : result;
+
+			if (soundContainer->GetCustomPanValue() != 0.0f) {
```

Comment:
What happens if you set the pan to, for example, 1, and then try to reset it to 0? Will it stay at 1?